package.json

```diff
@@ -67,6 +68,7 @@
     "emoji-datasource-apple": "4.0.0",
     "emoji-js": "^3.4.0",
     "emoji-panel": "https://github.com/scottnonnenberg/emoji-panel.git#v0.5.5",
+    "filesize": "^3.6.1",
```

Comment:
Note that `filesize` is a current Bower dependency, so we'll want to remove it from `bower.json` and `components`

Reply:
Oh, thanks for the heads-up, I didn’t notice that 👀

Reply:
Fixed.